src/components/user.js

```diff
@@ -203,17 +223,15 @@ export default class User extends React.Component {
                         disabled={!isFormFieldsValid}
                         onClick={this._handleClickSubmit}>Submit</button> :
                     (
-                        <button className="ButtonEdit"
-                                disabled={!isFormFieldsValid && editMode}
-                                onClick={editMode ? this._handleClickSave : this._handleClickEdit}>
-                            {editMode ? 'Save' : 'Edit'}
-                        </button>
+                        <Edit disabled={!isFormFieldsValid}
+                              onClickSave={this._handleClickSave}
+                              onClickEdit={this._handleClickEdit}
+                              onClickUndo={this._handleClickUndo}
+                              currentUser={currentUser}
+                              mode={mode}
```

Comment:
whould it be better to pass one prop `isEditing` to this component?
we will be able to remove `VIEWS` in it and just use conditional rendering because it's quite small component